libraries/freertos_plus/aws/greengrass/test/aws_test_greengrass_discovery.c

```diff
@@ -164,32 +141,6 @@ TEST( Full_GGD, JSONRequestAbort )
     /** @}*/
 }
 
-TEST( Full_GGD, prvIsIPvalid )
-{
-    BaseType_t xStatus;
-
-    if( TEST_PROTECT() )
-    {
-        /** @brief Check IP is correctly filtered if a loopback address is given.
-         *  @{
-         */
-        xStatus = test_prvIsIPvalid( ggdLOOP_BACK_IP, sizeof( ggdLOOP_BACK_IP ) );
-        TEST_ASSERT_TRUE( xStatus == pdFALSE );
-        /** @}*/
-
-        /** @brief Check IP is not filtered if correct IP is given.
-         *  @{
-         */
-        xStatus = test_prvIsIPvalid( clientcredentialMQTT_BROKER_ENDPOINT, sizeof( clientcredentialMQTT_BROKER_ENDPOINT ) );
-        TEST_ASSERT_TRUE( xStatus == pdTRUE );
-        /** @}*/
-    }
-    else
-    {
-        TEST_FAIL();
-    }
-}
-
 
 TEST( Full_GGD, GetGGCIPandCertificate )
```

Comment:
_Full__GGD seems misleading when considering the whole suite of tests for Greengrass. Using `GDD_System` would be appropriate now

Reply:
Changed